recipes/tket/patches/adjacency_list.diff

```diff
@@ -1,24 +0,0 @@
---- a/boost/graph/detail/adjacency_list.hpp
-+++ b/boost/graph/detail/adjacency_list.hpp
```

Comment:
I am a little confused: from my understanding of this PR this file should be deleted, but some lines are green?

Reply:
The file is deleted. I think that is just how github displays diff files.

Reply:
Ah, yes, thank you.